cmd/cmd.go

```diff
@@ -90,7 +91,7 @@ func ExecInCoreSnap() {
 	// ensure we do not re-exec into an older version of snapd, look
 	// for info file and ignore version of core that do not yet have
 	// it
-	fullInfo := filepath.Join(corePath, "/usr/lib/snapd/info")
```

Comment:
I will into this again. However all tests except refresh-delta-from-core on Ubuntu 14.04 seem to pass.